csharp/src/IceDiscovery/Locator.cs

```diff
@@ -10,16 +10,40 @@
 
 namespace ZeroC.IceDiscovery
 {
+    internal class Locator : ILocator
+    {
+        private readonly Lookup _lookup;
+        private readonly ILocatorRegistryPrx _registry;
+
+        public ValueTask<IObjectPrx?> FindAdapterByIdAsync(string adapterId, Current current) =>
+            _lookup.FindAdapter(adapterId);
```

Comment:
FindAdapter should be renamed FindAdapterAsync

Reply:
Fixed